services/svc/imds/src/package_2018_04_02/operations.rs

```diff
@@ -16,7 +16,7 @@ pub struct ClientBuilder {
     endpoint: Option<String>,
     scopes: Option<Vec<String>>,
 }
-pub const DEFAULT_ENDPOINT: &str = azure_core::resource_manager_endpoint::AZURE_PUBLIC_CLOUD;
+pub const DEFAULT_ENDPOINT: &str = "https://169.254.169.254/metadata";
```

Comment:
It is correct. https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service?tabs=windows